nodeadm/test/e2e/cases/kubeconfig/config.yaml

```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: node.eks.aws/v1alpha1
+kind: NodeConfig
+metadata:
+  name: example
```

Comment:
nit: this isn't necessary, right? we could omit this from all the test assets going forward

Reply:
true 👍